README.md

```diff
@@ -35,6 +35,15 @@ yarn add near-ca
 
 ## Usage
 
+Create a .env file. Make sure the environment variables are defined in the .env file. You can view the required variables in the .env.example
+
+Fund your address with some sepolia ETH
```

Comment:
This is a reasonable thing to insert, however the user may not be aware of the derived Ethereum account to fund. Maybe you could add a script like 'getEthAccount' and instruct the user to run it to determine the account that needs to be funded.